Tools/i18n/pygettext.py

```diff
@@ -373,7 +373,23 @@ def parse_spec(spec):
         raise ValueError(f'Invalid keyword spec {spec!r}: '
                          'msgctxt cannot appear without msgid')
 
-    return name, {v: k for k, v in result.items()}
+    return name, result
+
+
+def process_keywords(keywords, *, no_default_keywords):
+    custom_keywords_list = [parse_spec(spec) for spec in keywords]
+    custom_keywords = {}
```

Comment:
You can initialize it from DEFAULTKEYWORDS here. This will trivialize merging.

Reply:
I am not sure I can, custom keywords (`--keyword`) take priority over the default keywords so they way I do it now I need to first construct the custom keyword dictionary. I'd love to know if there's a way to simplify this though!

Reply:
Agree.